panther_battery/src/battery_node.cpp

```diff
@@ -48,27 +50,33 @@ BatteryNode::BatteryNode(const std::string & node_name, const rclcpp::NodeOption
 
   diagnostic_updater_->setHardwareID("Battery");
 
-  RCLCPP_INFO(this->get_logger(), "Node started");
+  RCLCPP_INFO(this->get_logger(), "Node constructed successfully.");
 }
 
 void BatteryNode::Initialize()
 {
+  RCLCPP_INFO(this->get_logger(), "Initializing.");
+
   const float panther_version = this->get_parameter("panther_version").as_double();
   if (panther_version >= (1.2f - std::numeric_limits<float>::epsilon())) {
     try {
       InitializeWithADCBattery();
       return;
     } catch (const std::runtime_error & e) {
-      RCLCPP_WARN_STREAM(this->get_logger(), "Failed to initialize ADC Battery: " << e.what());
-      RCLCPP_INFO(this->get_logger(), "Using Roboteq drivers to publish battery data.");
+      RCLCPP_WARN_STREAM(
+        this->get_logger(), "An exception ocurred while initializing with ADC: "
+                              << e.what()
+                              << " Falling back to using Roboteq drivers to publish battery data.");
     }
   }
   InitializeWithRoboteqBattery();
+
+  RCLCPP_INFO(this->get_logger(), "Initialized successfully.");
 }
 
 void BatteryNode::InitializeWithADCBattery()
 {
-  RCLCPP_INFO(this->get_logger(), "Initializing battery node using ADC data");
+  RCLCPP_DEBUG(this->get_logger(), "Initializing with ADC battery.");
```

Comment:
```suggestion
  RCLCPP_DEBUG(this->get_logger(), "Initializing with ADC data.");
```
ADC battery does not sound right to me. We use data from ADC to create the battery objects